doc/musig-spec.mediawiki

```diff
@@ -60,7 +62,120 @@ The following conventions are used, with constants as defined for [https://www.s
 ** The function ''point(x)'', where ''x'' is a 32-byte array ("x-only" serialization), returns ''lift_x(int(x))''. Fail if ''lift_x'' fails.
 ** The function ''pointc(x)'', where ''x'' is a 33-byte array (compressed serialization), sets ''P = lift_x(int(x[1:33]))'' and fails if that fails. If ''x[0] = 2'' it returns ''P'' and if ''x[0] = 3'' it returns ''-P''. Otherwise, it fails.
 ** The function ''hash<sub>tag</sub>(x)'' where ''tag'' is a UTF-8 encoded tag name and ''x'' is a byte array returns the 32-byte hash ''SHA256(SHA256(tag) || SHA256(tag) || x)''.
+* Other:
+** Tuples are written by listing the elements within parentheses and separated by commas. For example, ''(2, 3, 1)'' is a tuple.
+
+=== Tweaking ===
+
+Input:
+* ''P'': a point
+* The tweak ''t'': an integer with ''0 &le; t < n ''
+
+The algorithm ''OrdinaryTweak(P, t)'' is defined as:
+* Return P + t⋅G
+
+The algorithm ''XonlyTweak(P, t)'' is defined as:
+* Return set_even_y(P) + t⋅G
+
+=== Negation Of The Secret Key When Signing ===
+
+In order to produce a partial signature for an x-only public key that is an aggregate of x-only keys and x-only or ordinarily tweaked, the ''[[#Sign negation|Sign]]'' algorithm may need to negate the secret key during the signing process.
```

Comment:
I think it will be good to introduce u and v already in this sentence.

edit: I think this is related to my comment below that the sections could be reordered.

Reply:
done